### ISSUE TITLE
Abort host creation has two "Cancel" buttons that do different things in some localizations

### ISSUE THREAD

**Issue by callahad** [vector-im/element-web#16578]
![Screenshot of "abort host creation" dialog with two "Cancel" buttons](https://user-images.githubusercontent.com/24193/109551898-0bce0b80-7ac9-11eb-88db-6f58ecf34728.jpg)

Courtesy @jezdez at https://twitter.com/jezdez/status/1366416618398638082

As a QuestionDialog, this automatically gets a "Cancel" button which dismisses the dialog.

The dialog also uses an "Abort" button to mean "abort host creation"

Unfortunately, each of these are translated identically in German.

See: https://github.com/matrix-org/matrix-react-sdk/blob/v3.15.0/src/components/views/dialogs/HostSignupDialog.tsx#L123-L129

Weblate:

Abort: https://translate.element.io/translate/element-web/matrix-react-sdk/de/?offset=1&q=+source%3A%3DAbort+&sort_by=-priority%2Cposition&checksum=

Cancel: https://translate.element.io/translate/element-web/matrix-react-sdk/de/?offset=1&q=+source%3A%3DCancel+&sort_by=-priority%2Cposition&checksum=

**Comment by callahad** [comment]
Grepping reveals that the issue is present in:

- de_DE ("Abbrechen")
- fr ("Annuler")
- ir ("Annulla")
- nl ("Annuleren")
- ru ("Отмена")
- sv ("Avbryt)

Many languages do not yet have a string for "Abort", so I suspect the list will grow and that we should consider re-phrasing the dialogue to reduce the risk of this collision.

**Comment by jezdez** [comment]
Oh, thanks @callahad! FWIW I did try to search for the string on Weblate but for some reason didn't use the search correctly and gave up. Thank god for lazily reporting an issue on the interweb! 😬

**Comment by jezdez** [comment]
Oh now I realize why the search didn't work for me, I searched in the Element Web project, not the SDK. D'oh!

**Comment by jezdez** [comment]
"Verwerfen" would probably be a better term for "abort", it's often used in German UI translations to refer to discarding something. OTOH, I'm really wondering now if the whole dialog isn't misleading to be honest.

I came to that when I clicked on the "Upgrade to Element Home" menu item:

<img width="247" alt="CleanShot 2021-03-01 at 22 04 42@2x" src="https://user-images.githubusercontent.com/1610/109558811-28267400-7ada-11eb-9f21-23373404d2e0.png">

Then, after starting the signup process, it asked me to name my "home" and its "domain", but when I tried to cancel the process by closing the modal, it asked me about abourting the creation of a "host".

<img width="559" alt="CleanShot 2021-03-01 at 22 05 41@2x" src="https://user-images.githubusercontent.com/1610/109559007-6d4aa600-7ada-11eb-835f-731506c874ae.png">

I think there is an inconsistent UI language going on, and I think the dialog is kind of pointing to that a bit.
